src/cli.js

```diff
@@ -111,7 +117,7 @@ shouldRebuildPromise
   })
   .then((x, beforeRebuild) => {
     return installNodeHeaders(argv.v, argv.d, null, argv.a)
-      .then(() => rebuildNativeModules(argv.v, argv.m, argv.w, null, argv.a, argv.c))
+      .then(() => rebuildNativeModules(argv.v, argv.m, argv.w, null, argv.a, argv.c, argv.id, argv.io, argv.log))
```

Comment:
These one/two-letter variable names drive me nuts, can you use the aliases here instead? As in `argv.ignoreDevdeps`, `argv.ignoreOptdeps`. We can clean up the rest later.